src/components/form/Select/Select.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import { useStore } from 'context/Store';
+import Input from 'components/form/Input/Input';
+
+const Select = ({ register, errors, options, placeholder, type, setValue }) => {
+  const [isOpen, setisOpen] = useState(false);
+
+  const [, dispatch] = useStore();
```

Comment:
Since you want to call useStore but you don't want to use the state I think a better way to express that is like this:
```suggestion
  const [_state, dispatch] = useStore();
```

Reply:
I once saw in a react course that it was a good practice to leave it blank but i like this way you are suggesting to. Thanks